notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Is the time to merge in seconds?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
yes I'm pretty sure it seconds.  Timestamps of the form `1619104429` represent seconds since jan 1 1970